index.css

```diff
@@ -0,0 +1,199 @@
+*{
+      margin: 0px;
+      padding: 0px;
+}
+:root {
+      --primary-color: #F94892;
+      --secondary-color: #323232;
+      --black-color: hsl(0, 0%, 0%);
```

Comment:
Instead of plain black, you can use the shade of black.

Reply:
yes sure